juicebox_mqtthandler.py

```diff
@@ -342,7 +342,7 @@ async def set_mitm_handler(self, mitm_handler):
                 entity.add_kwargs(mitm_handler=mitm_handler)
 
     async def _basic_message_parse(self, data: bytes):
-        message = {"type": "basic"}
+        message = {"type": "basic", "current" : 0}
```

Comment:
```suggestion
        message = {"type": "basic", "current" : 0, "energy_session" : 0}
```

Should also add: "energy_session" : 0 as it also isn't going down to zero when the charging session ends.